Tests/Packet++Test/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@ add_executable(
   Tests/RadiusTests.cpp
   Tests/SipSdpTests.cpp
   Tests/SllNullLoopbackTests.cpp
+  Tests/Sll2Tests.cpp
```

Comment:
For some reason `cmake-format` fails and claims that `Tests/SllNullLoopbackTests.cpp` should come after `Tests/Sll2Tests.cpp`...

https://github.com/seladb/PcapPlusPlus/actions/runs/5617676562/job/15222130805?pr=1138

Reply:
done